### PR TITLE
Update funding

### PR DESCRIPTION
**Pull request by loftusa**
do not merge yet.
Making PR to have public access to Netlify on push.

### CODE REVIEW

**Comment by falkben** [comment]
FYI, netlify settings changed and prior build previews are gone.  make a new commit for a new build preview.

**Comment by loftusa** [comment]
ok, should be good to merge.
I'm not particularly satisfied with it right now -- I still need sentences for most of the grants, and there isn't much meat

but I think it's decent enough to go up, and we can make improve iteratively

File: content/about/funding.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+$title@: Funding
+$order: 7
+---
+
+This page lists currently active funding for our projects
+
+#### Project Title: Synaptomes of Mouse and Man
```
Comment (falkben) [inline content/about/funding.md]: remove "Project Title" from each of the headings.

File: content/about/funding.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+**R01NS092474 (Smith)**: 9/30/2014 – 6/30/2019
```
Comment (falkben) [inline content/about/funding.md:9]: we should probably add a link to each of the funding pages for each of these grants.

for instance, for synaptomes of mouse and man, http://grantome.com/grant/NIH/R01-NS092474-01

note, the synaptomes grant is about to end.

**Review by falkben — CHANGES_REQUESTED** [review]
Nice.

The PDF you are replacing with this has a "pending" category, which this page does not.  Are we adding that?

I think we might want a "past" category (or separate page).  See Slack #website discussion for that.

**Comment by falkben** [comment]
Mobile nav still has a link to old funding page

File: content/about/funding.md (hunk 1 above)
Comment (loftusa) [reply content/about/funding.md]: Will do

File: content/about/funding.md (hunk 2 above)
Comment (loftusa) [reply content/about/funding.md:9]: Cool, will do. I can add a hyperlink into either the grant title or the project title.

Comment (falkben) [reply content/about/funding.md:9]: not urgent

File: partials/about_nav.html [hunk 3]
```diff
@@ -14,7 +14,6 @@
                 <a href="{{item.url.path}}"><h5>{{item.title}}</h5></a></li>
             </li>
         {% endfor %}
-        <li><a href="{{g.static('/source/docs/CP_Vogelstein.pdf').url.path}}" target="_blank" rel="noopener"><h5>Funding</h5></a></li>
```
Comment (falkben) [inline partials/about_nav.html:17]: this line delete: https://github.com/neurodata/neurodata.io/pull/245/files#diff-dad88da953fd09c2c8b366b95e4c44b3R24

File: content/about/funding.md (hunk 2 above)
Comment (loftusa) [reply content/about/funding.md:9]: ok, to do later then. Will add to the issue.

File: partials/about_nav.html (hunk 3 above)
Comment (loftusa) [reply partials/about_nav.html:17]: done

File: content/about/funding.md (hunk 1 above)
Comment (loftusa) [reply content/about/funding.md]: done

**Comment by loftusa** [comment]
ok @falkben !

- removed "Project Title:"s
- per @jovo , did not add "overlap" or "pending" information
- fixed mobile nav bug

for future commit:
- did not add "past" information (not sure what the verdict is on that? @jovo), but could do so.
- link to funding pages on grantome
- need rest of sentences
- add neurodata logo or something to make it look nicer
- potentially add link to CP_vogelstein.pdf somewhere if people need extra information

CP_vogelstein.pdf is still in `source/docs` in case we need it

**Review by falkben — APPROVED** [review]
Thanks looks much better

**Pull request merged**